src/js/letters/utils/helpers.jsx

```diff
@@ -339,3 +339,37 @@ export function resetDisallowedAddressFields(address) {
 
   return newAddress;
 }
+
+/**
+ * Traverses a single-level object and removes its zero-length own-enumerable properties
+ * @param {Object} input an object with no nested properties
+ * @returns a cloned object with no empty properties
+ */
+export const stripEmpties = (input) => {
+  const newObject = { ...input };
```

Comment:
I just [confirmed on jsbin](http://jsbin.com/qifequzure/edit?js,console) that passing variables to a function does a shallow clone, so this line doesn't actually do anything. If you're looking to not modify `input`, you'll want to `_.deepClone()`.

Reply:
Er, I stand corrected. Passing happens by reference, so cloning is necessary.

This is still a shallow clone, but that's probably fine since the address passed as `input` isn't going to have any sub-objects.

Reply:
Yeah, I still don't think it's super great that this fact isn't actually enforced by the code, but don't think we need to do anything about it right now.